### ISSUE TITLE
Radio group: Contrasts insufficient (according to WCAG SC 1.4.11)

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria-practices#1481]
Examples:
* https://w3c.github.io/aria-practices/examples/radio/radio-1/radio-1.html 
* https://w3c.github.io/aria-practices/examples/radio/radio-2/radio-2.html

Problems:
* Contrast of an unselected radio button: 2.4:1
* Contrast of the focus frame: 2.3:1

Should be at least 3:1 in both cases ([WCAG SC 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html))

Additional problems in High Contrast Mode (Windows):
* Focus indicator not visible at all
* selected checkbox badly visible (only in Chrome and Edge, ok with Firefox): the contrast of the blue gradient is partly below 3:1 to the white background
